messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumer.java

```diff
@@ -88,6 +85,10 @@
   public static String clusterNamePrefix = "databusCluster";
   private Boolean readFromLocalStream;
   private int numList = 0;
+  public static List<String> collectorsWithReaders = new ArrayList<String>();
```

Comment:
@sreedishps  There can be easily multiple instances of MessagingConsumer created in the same VM. Hence don't have a static class member.